test-integration/workflow-samples/react.json

```diff
@@ -15,7 +15,7 @@
     {
       "name": "react-gradle-mysql-es-noi18n-mapsid",
       "app-sample": "react-gradle-mysql-es-noi18n-mapsid",
-      "entity": "sqlfull",
+      "entity": "sql",
```

Comment:
This is the only sqlfull test at react.

Reply:
Not passing with es: can I add a new sqlfull with react and without es?

Reply:
Just throwing up an idea there before creating the ticket while we are discussing about one use case, what about?
- First, add a one to one and mapsid in sqllight  (which are missing and impacting the backend queries and search)

- Making a default for all the frontend (sqlfull + Filters, h2-mem) which are impacting navigation on the frontend side
- Use sqllight and  split combinations of cache+build techs in fronts (only one combination should suffice, as combination does not impact front.
- Add 1 test of ES by front
- Add only 1 websocket and 1 messaging (no matter where, on the thinner one because it does not impact anything)
- Only one combination of build + Nosql split into front

So that we can basically cover all the features and reducing the build time by avoiding overlap on our tests.
WDYT?

Reply:
I have a issue or a pr closed somewhere defining some test matrix.